### DISCUSSION TITLE
debug exporter does not work?

### DISCUSSION THREAD

**Discussion by johnvlittle** [open-telemetry/opentelemetry-collector#11224] [General]
We integrated the debug exporter using this:

```
exporters.debug.verbosity = detailed
exporters.debug.sampling_initial = 5
exporters.debug.sampling_thereafter = 10
exporters.debug.use_internal_logger=true

```
And assumed that we would see some opentelemetry debug info in our standout or stderr, but nothing happens.  Where does the debug info go?

We also tried exporters.debug.use_internal_logger=false, but again no output.

we are using the auto-agent jar from here: https://opentelemetry.io/docs/zero-code/java/agent/getting-started/

**Comment by anjiemerchant** [comment]
i am also seeing this behavior
